examples/minimal-mdns/client.cpp

```diff
@@ -151,7 +152,7 @@ OptionDef cmdLineOptionsDef[] = {
     { "query-port", kArgumentRequired, kOptionQueryPort },
     { "timeout-ms", kArgumentRequired, kOptionRuntimeMs },
     { "multicast-reply", kNoArgument, kOptionMulticastReplies },
-    nullptr,
```

Comment:
Is this intended?

Reply:
this is pulling in a build fix from PR 6082, should go away when resolving conflicts...

Reply:
Still here, but maybe gone post conflicts...